app/models/user.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
+    'user',
+    {
+      name: DataTypes.STRING,
+      last_name: DataTypes.STRING,
```

Comment:
they should be in camelCase in the model

Reply:
But in the tech-guides they said that conventions the database and columns can be in snake_case